### DISCUSSION TITLE
v2 upgrade source

### DISCUSSION THREAD

**Discussion by adrianrudnik** [kimai/kimai#3890] [Questions & Answers] [answered]
Nice to see v2 working and near release now! Just wanted to ask what the best source is to use? I see the main branch and  `release-x.x.x`-branches floating around, but nothing that fixed that can be chosen as fixed "tags" to version upgrades. In 1.x I could just shift the git clone to the new version tag, now I'm not sure what to use. Any recommendations?

**Comment by kevinpapst** [comment, marked as answer]
https://github.com/kimai/kimai/tags

**Reply by adrianrudnik** [reply]
Oh boy, did not see them, maybe coming from the release tab. Thanks!

**Reply by kevinpapst** [reply]
no release on purpose yet 😁 
too many support questions that will pop-up, and I am trying to avoid that for a couple of weeks, while fixing the obvious day-one bugs
